auth_provider/templates/auth/login.html

```diff
@@ -1,23 +1,108 @@
-{% extends "base.html" %} {% block content %} {% if form.errors %}
-<div class="alert">
-  <p>Username and password did not match</p>
-</div>
-{% endif %}
+{% extends "base.html" %} {% block content %} 
+
+<style>
+  body {
+    font-family: Arial, Helvetica, sans-serif;
+  }
+
+  * {
+    box-sizing: border-box;
+  }
+
+  /* Add padding to containers */
+  .container {
+    padding-top: 20px;
+    padding-right: 50px;
+    padding-bottom: 20px;
+    padding-left: 50px;
+    background-color: white;
+  }
+
+  /* Full-width input fields */
+  input[type=text], input[type=password] {
+    width: 100%;
+    padding: 15px;
+    margin: 5px 0 22px 0;
+    display: inline-block;
+    border: none;
+    background: #f1f1f1;
+  }
+
+  input[type=text]:focus, input[type=password]:focus {
+    background-color: #ddd;
+    outline: none;
+  }
+
+  /* Overwrite default styles of hr */
+  hr {
+    border: 1px solid #f1f1f1;
+    margin-bottom: 25px;
+  }
 
-<h1>Login</h1>
-<form method="post" action="{% url 'auth_provider:login' %}">
-  {% csrf_token %}
+  /* Set a style for the submit button */
+  .registerbtn {
+    background-color: #2b73e7;
+    color: white;
+    padding: 16px 20px;
+    margin: 8px 0;
+    border: none;
+    cursor: pointer;
+    width: 100%;
+    opacity: 0.9;
+  }
```

Comment:
I think this is better as primary/secondary buttons in `styles.css`
E.g. 
```css
.primaryButton {
}
.secondaryButton {
}
```
or even
```css
.button {
}
.button.primary {
}
.button.secondary {
}
```